Tools/WinMLDashboard/src/view/edit/LeftPanel.tsx

```diff
@@ -70,32 +72,23 @@ class LeftPanel extends React.Component<IComponentProperties, {}> {
     public render() {
         return (
             <div className="Unselectable">
-                <Resizable visible={this.props.selectedNode !== undefined}>
-                    {this.props.selectedNode !== undefined && this.getContent()}
+                <Resizable visible={this.props.showLeft}>
+                    {this.props.showLeft && this.getContent()}
                 </Resizable>
             </div>
         );
     }
 
     private getContent() {
-        let name = 'Model Properties';
-        const modelPropertiesSelected = this.props.selectedNode === name;
-        let input: any[];
-        let output: any[];
-        if (modelPropertiesSelected) {
-            input = this.props.modelInputs;
-            output = this.props.modelOutputs;
-        } else {
-            const node = this.props.nodes[this.props.selectedNode];
-            ({ input, output, name } = node);
-            name = `Node: ${name ? `${name} (${node.opType})` : node.opType}`;
-        }
+        const name = 'Input And Output';
 
+        const input = this.props.modelInputs;
+        const output = this.props.modelOutputs;
         const inputsForm = this.buildConnectionList(input);
         const outputsForm = this.buildConnectionList(output);
         return (
             <div>
-                <Label className='PanelName'>{name}</Label>
+                <Label className='PanelName' onClick={this.toggleLeft}>{name}</Label>
```

Comment:
This is not be needed if "Edit" button toggles between "View Only" mode and "Edit" mode. View Only mode gives the graph browsing experience. Edit mode should allow both editing and browsing (browsing without Netron's flyout).

The "Edit" button (next to zoom buttons) should be the only place to toggle from Edit mode to View only mode. Currently, just touching any node in the graph switches edit mode to view only mode which makes using the edit mode a bit cumbersome (as mere browsing the graph can lead to switching the mode inadvertently).